### DISCUSSION TITLE
Using Epoptes on a thin client

### DISCUSSION THREAD

**Discussion by misiek428** [ltsp/ltsp#414] [Q&A] [answered]
Hi,
I installed the LTSP server on Ubuntu Mate 18.04. Users use thin clients, log into the server - everything works fine.
I installed Epoptes on the server, created a new user and added it to Epoptes:
  
gpasswd -a **new_user** epoptes

If I log into the **new_user** account on the server, I use Epoptes without any problems.

I logged on to the thin client computer with the **new_user** account. I wanted to use Epoptes, but it is not in:

Menu -> Internet -> Epoptes. 

I tried via terminal but there is only epoptes-client.

My question is - can I use Epoptes from a different machine than the server?

Thank you for your help!

**Comment by alkisg** [comment]
Hi, see the documentation about this: https://epoptes.org/documentation/run-fat/

For epoptes discussions, it's best to use: https://github.com/epoptes/epoptes/discussions

**Reply by misiek428** [reply]
Thank you, I'll be looking there.

**Comment by Lichtmacher** [comment, marked as answer]
I use this:
ssh -X user@SERVER epoptes

**Reply by misiek428** [reply]
I tried it and it works.

Thank you very much for your help :)
